### ISSUE TITLE
Add an option to retrieve commits modifying a deleted file

### ISSUE THREAD

**Issue by 12f23eddde** [ishepard/pydriller#199]
Hi :) I'm extracting the commit history of a certain configuration file on thousands of public repos hosted on Github. Some projects have this file deleted at some point, making it impossible to do so. 

It would be nice to add an argument to the `Repository` class, which inserts `--` to the `git log` call:
```python
for commit in Repository(repo
     filepath=file_to_look_for,
     full_log=True).traverse_commits():
```

This feature is actually quite easy to implement [reference here](https://feeding.cloud.geek.nz/posts/querying-deleted-content-in-git/), [my fork](https://github.com/12f23eddde/pydriller/commit/9a44e7f5526c4dec4f43ed8d3cabf34cfa1254b4) made just a little change to get it working:

```python
def get_commits_modified_file(self, filepath: str, full_log=False) -> List[str]:
      if full_log:
          commits = self.repo.git.log("--follow", "--format=%H", "--", path).split('\n')
      else:
          commits = self.repo.git.log("--follow", "--format=%H", path).split('\n')
```

I would like to open a pull request if this change sounds reasonable.

**Comment by ishepard** [comment]
Hi @12f23eddde! 
Yes sure this change makes total sense! Feel free to make a PR.

Instead of `full_log` I would say to call it `include_deleted_files`, just to make it even more clear what's for.

**Comment by 12f23eddde** [comment]
Sorry I've been busy the last few weeks, and #201 is already working on this. Guess this issue can be closed (when #201 finishes the task)?

**Comment by ishepard** [comment]
Yes I will close the task when the PR is accepted 😃
